Fix donokuda hire post typos (#15)
aseriousbiz/blog: _posts/2021/2021-09-13-don-okuda-is-an-abboteer.md

    name: pmn
---

> We’re excited to share that @niik and @joshaber have put Don back in the dryer and Don will be delivered back to GitHub ASAP.

Reviewer: ```suggestion
We’re excited to share that @niik and @joshaber have also joined A Serious Business to help Don get out of the dryer and also because they miss working with @haacked, @shana, and @pmn, and they love working on bots.
```

> ![Mad Don](https://user-images.githubusercontent.com/13760/133258348-41e83378-fb4d-4e3c-9ec3-4167a25b0522.png)

Reviewer: 😹 
<img width="1034" alt="and now it's in Abbot" src="https://user-images.githubusercontent.com/279389/133277691-ee7b1363-7950-4307-8d55-04cb57ce38ee.png">

> We all worked with Don at GitHub on GitHub Desktop and can’t wait for him to go back to working on it!

Reviewer: ```suggestion
We all worked with Don at GitHub on GitHub Desktop and can’t wait for Josh and Markus to work with us too!
```

> When he’s not making great products, Don makes music and enjoys pineapple on his pizza.

Reviewer 2: +1 this line change 🚀 :shipit:

> Don insists he’s an actual human, so please join us in welcoming him back to GitHub!

Reviewer: ```suggestion
Don insists he’s an actual human, so please join us in welcoming him, Markus, and Josh to A Serious Business!
```
